doc-content/enterprise-only/getting-started-kogito/proc-kogito-modelers-install.adoc

```diff
@@ -0,0 +1,38 @@
+[id="proc-kogito-modelers-install_{context}"]
+=== Installing and using the {KOGITO} Business Modeler Hub
+
+{KOGITO} provides a https://kiegroup.github.io/kogito-online/#/download[Business Modeler Hub] desktop application with all extensions or applications for modeling Decision Model and Notation (DMN) decision models in {KOGITO} microservices.
+
+As you develop {KOGITO} microservices, you can use the {KOGITO} Business Modeler Hub to install, launch, or uninstall the available modeling extensions or applications as needed.
+
+.Procedure
+. In a web browser, go to the https://kiegroup.github.io/kogito-online/#/download[Business Modeler Hub] download page, select the relevant *Operating System* option, and click *Download*.
+. Extract the downloaded `business_modeler_hub_preview___RELEASE__` binary file to a local directory.
```

Comment:
[style] Is `business_modeler_hub_preview___RELEASE__` correct with the additional underscores? Can the version attribute be used?

Reply:
We are not using attribute here, this is the original file name with Release in italics to show that release version needs to be added by the user.